### ISSUE TITLE
Verify is not evaluating the profile components

### ISSUE THREAD

**Issue by smartinez290** [haffonist/igamt#2724]
Verify available at the segment, datatype and conformance profile message level, but is  not available at the profile component level. The "Verify" in the main tab do not evaluate the profile component either. I am not sure if this is the original intention.

**Comment by Abdelghani90** [comment]
I don't think this is a bug. We never implemented a validation for profile component. I think this need more discussion
